DSESwift/QueryOptions.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  queryOptions.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 01/10/15.
+//
+//
+
+import Foundation
+
+public struct QueryOptions{
+    
+    public enum Interval {
+        case Minute
+        case Hour
+        case Day
+        case Week
+        
+        var description : String {
+            switch self {
+                // Use Internationalization, as appropriate.
+                case .Minute: return "Minute";
+                case .Hour: return "Hour";
+                case .Day: return "Day";
+                case .Week: return "Week";
+            }
+        }
+    }
+
+    var startDate : NSDate?  // null by default, if not null, change from default
+    var endDate: NSDate? // null by default, if not null, change from default
+    var existsInCS: Bool? // null by default, if not null, change from default
+    var limit: Int?
+    var sortOrder: SortOrder
+    var interval:Interval?
+    
+    public init(startDate: NSDate?, endDate: NSDate?, existsInCS: Bool?, limit: Int?, sortOrder: SortOrder, interval: Interval?) {
```

Comment:
@kanekin doesn't the struct type already provide Memberwise Initializers so we don't need to define an explicit init anymore?